scripts/travisci/check_pylint.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+pylint --disable=all --enable=C0301,C0330 $(git diff HEAD HEAD~1 --name-only | grep "\.py")
```

Comment:
can you add the errors/ignore and incremental vs absolute features to this file just like flake8? that will make them easy to maintain.

Reply:
Since error codes C0301 C0330 are for indentation but the yapf corrections for indentation conflict with pylint, I think it's better to remove it. Other checks now can be done with flake8 thanks to the support packages for docstring and naming conventions.